presto-docs/src/main/sphinx/security/hive.rst

```diff
@@ -0,0 +1,241 @@
+                                                   metastore.
+
+``hive.metastore.presto.principal``                The Kerberos principal that Presto will use when connecting
+                                                   to the Hive metastore.
+
+``hive.metastore.presto.keytab``                   The path to the keytab file that contains a key for the
+                                                   principal specified by ``hive.metastore.presto.principal``.
+
+================================================== ============================================================ ==========
+
+``SIMPLE`` authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+    hive.metastore.authentication.type=SIMPLE
+
+``SIMPLE`` authentication is the default authentication mechanism for the Hive
+metastore. When using ``SIMPLE`` authentication, Presto connects to an
+unsecured Hive metastore. Kerberos is not used.
+
+``SASL`` authentication
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+    hive.metastore.authentication.type=SASL
+    hive.metastore.principal=hive/hive-metastore-host.example.com@EXAMPLE.COM
+    hive.metastore.presto.principal=presto@EXAMPLE.COM
+    hive.metastore.presto.keytab=/etc/presto/hive.keytab
+
+When using ``SASL`` authentication for the Hive metastore Thrift service,
+Presto will connect as the principal specified by the property
+``hive.metastore.presto.principal``.  Presto will authenticate this principal
+using the keytab specified by the ``hive.metastore.presto.keytab`` property,
+and will verify that the identity of the metastore matches
+``hive.metastore.principal``.
+
+Keytab files must be distributed to every node in the cluster that runs Presto.
+
+:ref:`Additional information on keytab files.<hive-security-additional-keytab>`
+
+Hive HDFS Authentication
+------------------------
+
+In a Kerberized Hadoop cluster, Presto authenticates to HDFS using Kerberos.
+Kerberos authentication for HDFS is configured in the connector's properties
+file using the following properties:
+
+================================================== ============================================================ ==========
+Property Name                                      Description                                                  Default
+================================================== ============================================================ ==========
+``hive.hdfs.authentication.type``                  One of ``SIMPLE`` or ``KERBEROS``. When set to ``KERBEROS``, ``SIMPLE``
+                                                   the Hive connector authenticates to HDFS using Kerberos.
+                                                   When using the default value of ``SIMPLE``, Kerberos
+                                                   authentication is disabled and no other properties need to
+                                                   be configured.
+
```

Comment:
Fixed. I split the sentences into paragraphs too. Let me know what you think, please.